ui/src/App.svelte

```diff
@@ -16,15 +16,15 @@
   } from "./api";
 
   onMount(() => {
-    localStorage.clear();
+    // localStorage.clear();
```

Comment:
Why remove this?

Reply:
it keep removing data after every refresh.